HW04/Kate Fedotova/array_spec.rb

```diff
@@ -0,0 +1,45 @@
+require 'rspec'
+require 'rubocop-rspec'
+require_relative 'arr.rb'
+# rubocop:disable Metrics/BlockLength
+describe Array do
+  describe '#my_each' do
+    subject(:arr) { [1, 2, 3, 4] }
+
+    context 'when has block' do
+      it 'return arr items +1'
+      new_arr = []
+      arr.my_select { |el| new_arr << el }
+      expect(new_arr).to eq([1, 2, 3, 4])
```

Comment:
Shouldn't we use this way? 🤔 
```suggestion
      it 'return arr items +1' do 
        new_arr = []
        arr.my_select { |el| new_arr << el }
        expect(new_arr).to eq([1, 2, 3, 4])
      end
```